src/tensorwaves/data/generate.py

```diff
@@ -12,64 +14,58 @@
 )
 from tensorwaves.interfaces import (
     Function,
-    Kinematics,
     PhaseSpaceGenerator,
     UniformRealNumberGenerator,
 )
-from tensorwaves.physics.helicity_formalism.kinematics import (
-    HelicityKinematics,
-    ParticleReactionKinematicsInfo,
-)
 
 
 def _generate_data_bunch(
     bunch_size: int,
     phsp_generator: PhaseSpaceGenerator,
     random_generator: UniformRealNumberGenerator,
     intensity: Function,
-    kinematics: Kinematics,
-) -> np.ndarray:
+    kinematics: HelicityKinematics,
```

Comment:
This should be a Kinematics interface. If we do not have an interface for that, then one has to be created

Reply:
Fixed by 6b8a559 and follow-up commits